### PR TITLE
[FEATURE] Document new events

### PR DESCRIPTION
**Pull request by linawolf**
resolves https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/121
resolves https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/122
resolves https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/124

### CODE REVIEW

File: Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst [hunk 1]
```diff
@@ -137,32 +131,27 @@ which is the point where the link is handed over to the LinkBrowser for further
...
-   May modify the list of available LinkHandlers and has to return the final list.
+Supporting both TYPO3 12 and 11 to modify the available LinkHandlers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-modifyAllowedItems(allowedTabs, currentLinkParts)
-   May modify the list of available tabs and has to return the final list.
+If you want to be compatible to both TYPO3 12 and 11 you can keep your
+implementation of the hooks
+:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['LinkBrowser']['hooks']` as
+described in :ref:`t3coreapi11:modifyLinkHandlers`. And implements the
```
Comment (brotkrueml) [inline Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst:155]:
```suggestion
described in :ref:`t3coreapi11:modifyLinkHandlers` and implement the
```

File: Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst [hunk 2]
```diff
@@ -137,32 +131,27 @@ which is the point where the link is handed over to the LinkBrowser for further
...
+*  :ref:`ModifyLinkHandlersEvent`
 
-modifyLinkHandlers(linkHandlers, currentLinkParts)
-   May modify the list of available LinkHandlers and has to return the final list.
+Supporting both TYPO3 12 and 11 to modify the available LinkHandlers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-modifyAllowedItems(allowedTabs, currentLinkParts)
-   May modify the list of available tabs and has to return the final list.
+If you want to be compatible to both TYPO3 12 and 11 you can keep your
```
Comment (brotkrueml) [inline Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst:152]:
```suggestion
If you want to be compatible to both TYPO3 12 and 11, you can keep your
```

**Review by brotkrueml — DISMISSED** [review]

File: Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst [hunk 3]
```diff
@@ -137,32 +131,27 @@ which is the point where the link is handed over to the LinkBrowser for further
...
-   :caption: EXT:site_package/ext_tables.php
+Events to modify the available LinkHandlers
+--------------------------------------------
 
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['LinkBrowser']['hooks'][1444048118] = [
-       'handler' => \Vendor\Ext\MyClass::class,
-       'before' => [], // optional
-       'after' => [] // optional
-   ];
+You may have the need to modify the list of available LinkHandlers based on
```
Comment (sypets) [inline Documentation/ApiOverview/LinkBrowser/LinkBrowserApi/Index.rst:141]:
```suggestion
You may have to modify the list of available LinkHandlers based on
```

**Review by sypets — CHANGES_REQUESTED** [review]
Minor suggestion

**Review by sypets — APPROVED** [review]

**Pull request merged**